terraform_files/vsphere/main.tf

```diff
@@ -91,6 +91,7 @@ resource "vsphere_virtual_machine" "master" {
   guest_id                    = "coreos64Guest"
   folder                      = var.vsphere_folder != "" ? "${var.vsphere_parent_folder}/${local.folder}" : vsphere_folder.folder[0].path
   enable_disk_uuid            = "true"
+  hardware_version            = 15
```

Comment:
what's the default?

Reply:
That is baked in OVA which we use, so we start with HW 14, run 4.11 CSI tests and these complain that you need at least 15 - see `[sig-arch][Early] Managed cluster should start all core operators` test message in this [test run](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-assisted-test-infra-master-e2e-vsphere-assisted-periodic/1572012854410743808)